lib/rules/newline-before-return.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
             recommended: false
         },
 
+        fixable: "whitespace",
+
```

Comment:
Looks like you were just keeping it consistent with what was already there, but I think we can get rid of the extra newlines between props here